### DISCUSSION TITLE
Feature request

### DISCUSSION THREAD

**Discussion by infromthecold** [maziac/DeZog#47] [Ideas]
Hi Maziac
              Thanks for the great work, the latest update is very nice, making life easier, i was wondering of you could add a feature, I have found i am constantly moving or reducing the registers view window. Therefore i wondered if you could put it on a button on the debugger menu rather than opening it every time we start our code as many times i don't need that window especially as all the registers come in the variables window at a breakpoint. Yes i know that window is very useful as it is showing the memory registers are pointing too, however as I say it gets in my way most of the time. 
              
Thanks for the great work

Patricia

**Comment by maziac** [comment]
Thanks for your request.

Just to be sure: are you talking of this view:
![Bildschirmfoto 2021-02-17 um 18 00 14](https://user-images.githubusercontent.com/3081973/108239269-ff9d9200-7149-11eb-9ccd-e29116adb016.jpg)

What I could do is add an option to launch.json to hide it initially.

But there is no way to bring it back at the moment.
If you have a good idea how to do it please let me know. Could you elaborate on the "button on the debugger menu". Which menu are you referring to?

I could, of course, add a command to the debug console or even to the command palette. But that is not as easy as pressing a button.

**Reply by infromthecold** [reply]
Yes that window, yes that launch.json would be a good solution as you could select a debugging session, like you would switching between cspect and real next hardware, using a command switch would mean we could debug with the window, or debug without it using something like "registerMemoryWindow":true or false  

I think the debugging buttons i mentioned are part of VS code, so that would not work.

**Comment by maziac** [comment]
There is a beta version available now: 
https://github.com/maziac/DeZog/releases/tag/v2.1.7-beta
Please let me know if it's OK for you.

**Reply by infromthecold** [reply]
Many thanks i will look at it!

**Reply by infromthecold** [reply]
Many thanks for this, worked opposite to how i expected as i expected the -rmv ro remove the window rather than give the window , but hey no matter.

Great stuff, the kids are going to love this.

**Comment by maziac** [comment]
The 'r' stands for 'register' (memory view) not for 'remove' :-)
